controllers/usersignup/masteruserrecord_test.go

```diff
@@ -222,6 +224,41 @@ func TestMigrateMurIfNecessary(t *testing.T) {
 			assert.True(t, changed)
 			assert.Equal(t, newExpectedMur(nsTemplateTier, userSignup), mur)
 		})
+
+		t.Run("when tierName is missing", func(t *testing.T) {
+			userSignup := NewUserSignup()
+			nsTemplateTier := newNsTemplateTier("advanced", "dev", "stage", "extra")
+			mur, err := newMasterUserRecord(userSignup, test.MemberClusterName, nsTemplateTier, "johny")
+			require.NoError(t, err)
+			mur.Spec.TierName = ""
+			mur.Spec.UserAccounts[0].Spec.NSTemplateSet = &toolchainv1alpha1.NSTemplateSetSpec{}
```

Comment:
may I know why there is this line?
```
 			mur.Spec.UserAccounts[0].Spec.NSTemplateSet = &toolchainv1alpha1.NSTemplateSetSpec{}
```
it shouldn't be related to the test case, right?
```suggestion
```

Reply:
> may I know why there is this line?

no 😂